pyproject.toml

```
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
     'numpy>=1.22',
-    'scipy',
+    'scipy<1.18.0',
```

If we do this, we will always get the deprecation warnings, alternatively we could do
```suggestion
    'scipy<1.15.0',
```
to avoid this. Note that there might currently be a scipy bug, that would not be fixed in 0.6.3: https://github.com/scipy/scipy/issues/23101